90.md

```diff
@@ -67,6 +67,37 @@ All tags are optional.
 * `relays`: List of relays where Service Providers SHOULD publish responses to
 * `p`: Service Providers the customer is interested in. Other SPs MIGHT still choose to process the job
 
+## Encrypted Params 
+
+If the user wants to keep the input parameters a secret, they can encrypt the `i` and `param` tags with the service provider's 'p' tag and add it to the content field. The user also indicates whether the output should be encrypted or not as one of the parameters.
```

Comment:
This should mention explicitly that this is following the NIP-04 encryption scheme. Just bring in the same text that NIP-51 has:

> Encryption for private tags will use [NIP-04 - Encrypted Direct Message](https://github.com/nostr-protocol/nips/blob/master/04.md) encryption, using the list author's private and public key for the shared secret